docs/access.rst

```diff
@@ -81,14 +83,17 @@ While **Kolibri** is up and running on the device where you installed it, other
 
 * You can check the IP (**Server URL**) of the device where Kolibri is running by going to **Info** tab in the **Device** dashboard, .
 
-    .. figure:: img/device-info.png
-      :alt: Find out the IP/Server URL in the Device > Info tab.
+  .. figure:: img/device-info.png
+    :alt: Find out the IP/Server URL in the Device > Info tab.
 
-      Find out the IP/Server URL in the Device > Info tab.
+    Find out the IP/Server URL in the Device > Info tab.
 
 .. note::
   You can also use the ``ipconfig`` command on Windows or ``ifconfig`` command on Linux/OSX to find out the IP address of the device running the **Kolibri**.
 
+.. tip::
+  You can also use the ``ipconfig`` command on Windows or ``ifconfig`` command on Linux/OSX to find out the IP address of the device running the **Kolibri**.
+
```

Comment:
The "tip" duplicates the "note" above it

Reply:
Forgot to delete... 😒